recipes/default.rb

```diff
@@ -80,6 +80,7 @@
     owner node.sensu.admin_user
     group node.sensu.group
     mode 0640
+    sensitive true if Chef::Resource::ChefGem.instance_methods(false).include?(:sensitive)
```

Comment:
@cbandy I think there were some issues with this and previous versions of Chef due to how the compile and run phases work. A similar patch was attempted when the `compile_time` attribute was introduced and the `respond_to?` method did not work.